www/barcodescanner.js

```diff
@@ -73,8 +73,8 @@
  * @param {Function} errorCallback
  * @param config
  */
-BarcodeScanner.prototype.scan = function (successCallback, errorCallback, config) {
-
+BarcodeScanner.prototype.scan = function (successCallback, errorCallback, force, config) {
```

Comment:
Can we add the option inside the config? also, with this change the documentation becomes outdated, we need to update that, also, is this a public library?